### PR TITLE
refix database connection, GUI Search Cart, Detal CartItem

### PR DESCRIPTION
**Pull request by pekka1112**

### CODE REVIEW

File: ComMeNau/src/main/java/com/commenau/connectionPool/ConnectionPool.java
Comment (nguyenngocquy21130506) [inline ComMeNau/src/main/java/com/commenau/connectionPool/ConnectionPool.java:1]: ?

File: ComMeNau/src/main/java/com/commenau/controller/customer/CartController.java
Comment (nguyenngocquy21130506) [inline ComMeNau/src/main/java/com/commenau/controller/customer/CartController.java:1]: ?

File: ComMeNau/src/main/java/com/commenau/dao/LogDao.java
Comment (nguyenngocquy21130506) [inline ComMeNau/src/main/java/com/commenau/dao/LogDao.java:1]: ?

File: ComMeNau/src/main/java/com/commenau/service/CartService.java
Comment (nguyenngocquy21130506) [inline ComMeNau/src/main/java/com/commenau/service/CartService.java:1]: ?

**Review by nguyenngocquy21130506 — APPROVED** [review]

**Review by QuanghuyDrawn — APPROVED** [review]

**Pull request merged**